src/sentry_scope.c

```diff
@@ -337,7 +338,8 @@ sentry__scope_apply_to_event(const sentry_scope_t *scope,
     sentry_value_decref(contexts);
 
     if (mode & SENTRY_SCOPE_BREADCRUMBS) {
-        PLACE_CLONED_VALUE("breadcrumbs", scope->breadcrumbs);
+        PLACE_CLONED_VALUE("breadcrumbs",
+            sentry__value_ring_buffer_to_list(scope->breadcrumbs));
```

Comment:
Do we need to clear the list after cloning it? Currently tests are failing, which might be because the old `scope->breadcrumbs` got cleared out anyway, but we now allocate a new list to clone instead.